### PR TITLE
Fix realtime, fix version check on first start.

### PR DESCRIPTION
**Pull request by jacquarg**
* Add test guide for human powered validation
* Fix realtime (fix files list update on changes)
* Fix messages on bad platform version during initialization process.
* Fix create photo folder, which were broken with the new routes.
* Add retries on initial replication get files and get folders steps, to hide view indexation.
* Fix check version algorithm
* Update min Proxy version
* Wait for data-system min version with https://github.com/cozy/cozy-data-system/commit/acdf87f46366351120a26e090f4610193c6ea7a6

### CODE REVIEW

File: www-src/app/replicator/main.coffee [hunk 1]
```diff
@@ -64,8 +64,11 @@ module.exports = class Replicator extends Backbone.Model
 
     checkPlatformVersions: (callback) ->
         cutVersion = (s) ->
-            [s, major, minor, patch] = s.match /(\d+)\.(\d+)\.(\d+)/
-            return { major, minor, patch }
+            parts = s.match /(\d+)\.(\d+)\.(\d+)/
+            # Keep only usefull data (first elem is full string)
```
Comment (nono) [inline www-src/app/replicator/main.coffee]: _useful_ (only 1 `l`)

File: www-src/test/assets/manual_test_guide.md [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+[ ] Add a file in cache
+[ ] Rename this file in cache
+[ ] Delete this file in cache
+[ ] Add a folder
+[ ] Add a folder in cache
+[ ] Add a file in the folder
+[ ] Change the file in the folder
+[ ] Delete the folder
+
+[ ] Delete Photos directory, and let the app recreate it.
```
Comment (nono) [inline www-src/test/assets/manual_test_guide.md]: The blank line should be after the line, not before. No ?

**Comment by poupotte** [comment]
@nono , @jacquarg Can I marge this PR ?

**Comment by jacquarg** [comment]
@poupotte  : good for me (I  hope ; )

**Comment by nono** [comment]
@poupotte it's merged

**Comment by poupotte** [comment]
@nono  Thanks :)

**Pull request merged**